tests/macsec/macsec_config_helper.py

```diff
@@ -125,9 +136,10 @@ def cleanup_macsec_configuration(duthost, ctrl_links, profile_name):
     for dut_port, nbr in ctrl_links.items():
         disable_macsec_port(duthost, dut_port)
         disable_macsec_port(nbr["host"], nbr["port"])
-        delete_macsec_profile(nbr["host"], nbr["port"], profile_name)
         devices.add(nbr["host"])
-        delete_macsec_profile(duthost, dut_port, profile_name)
+    delete_macsec_profile(nbr["host"], nbr["port"], profile_name)
```

Comment:
Why do you move this line out of the `for loop`? I think we should remove the profile in all neighbor devices.

Reply:
I have updated the approach for delete_macsec_profile() so that we call it once after the macsec profile is removed from all interfaces.